### PR TITLE
solution to a task

### PR DESCRIPTION
**Pull request by Timi2424**
https://Timi2424.github.io/layout_moyo-header/
https://Timi2424.github.io/layout_moyo-header/report/html_report/
https://monosnap.com/file/IsLs4HJ1f1v9ZjZ5qsfpe3GlAhunnf

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,44 @@
...
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <ul class="nav">
+        <img
```
Comment (mbruhler) [inline src/index.html:17]: Please refer to this [guideline](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping) and fix the formatting

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,44 @@
...
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <ul class="nav">
```
Comment (mbruhler) [inline src/index.html:16]: It's not using BEM, please use it

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
+  font-size: 12px;
+  border-bottom: 4px;
+  border-color: rgba(0, 172, 220, 1);
+}
+
+div {
+  background-color: aqua;
```
Comment (mbruhler) [inline src/style.css:46]: use color hex instead of name

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,48 @@
...
+  font-size: 12px;
+  font-weight: 500;
+  margin: 10px;
+  height: 15px;
+}
+
+.nav a:hover {
+  color: black;
+  text-decoration: underline ;
+  text-decoration-color: blue;
```
Comment (mbruhler) [inline src/style.css:30]: use color hex instead of name and move them to CSS Variables

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,48 @@
...
+}
+
+.logo {
+  size: 40px 40px;
```
Comment (mbruhler) [inline src/style.css:34]: move size also to css variables

**Review by mbruhler — CHANGES_REQUESTED** [review]
Hi,
Please add links to your test and github pages.

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,63 @@
...
+  background-color: white;
+  padding: 10px 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 40px;
+}
+
+.header__logo {
+  height: 40px;
```
Comment (mbruhler) [inline src/style.css:19]: Please move this to css variables

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,63 @@
...
+  width: 40px;
+}
+
+.header__img {
+  height: 40px;
```
Comment (mbruhler) [inline src/style.css:24]: Then you can use this variable here

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,63 @@
...
+
+.nav__link:hover,
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  display: block;
+  background-color: #00acdc;
```
Comment (mbruhler) [inline src/style.css:53]: Colors are still not in variables

**Review by mbruhler — CHANGES_REQUESTED** [review]
Your links still don't work. Please refer to documentation in the readme to solve this problem.

Also your new changes don't include my past requests. Please include them in the next update.

**Review by mvjl000 — CHANGES_REQUESTED** [review]
Hey, could you re-deploy your solution? Your DEMO and report is not working (404)

**Review by mbruhler — APPROVED** [review]
Good job!
Approved :)

**Pull request closed without merging**